osprey/processes/wps_convolution.py

```diff
@@ -75,13 +75,71 @@ def __init__(self):
             "complete": 100,
         }
         inputs = [
+            log_level,
             LiteralInput(
-                "convolve_config",
-                "Configuration",
-                abstract="Path to input configuration file or input dictionary",
+                "case_id",
+                "Case ID",
+                abstract="Case ID for the RVIC process (required)",
```

Comment:
Aren't these abstracts shown in `bird?`. I thought it would give a better direction of using the birds.

Reply:
Yes they are shown and yes it would probably provide more direction. I was just pointing out that the bird would complain if there was any item missing.